examples/src/main/scala/Server.scala

```diff
@@ -27,21 +27,16 @@ object errors {
   }
 }
 
-object Server extends App {
+object Server extends App with RouterDerivationMacro {
   import controllers._
   import wiro.reflect._
   import models._
   import errors._
   import FailSupport._
 
-  val doghouseApi = new DoghouseApiImpl
+  val doghouseApi = new DoghouseApiImpl: DoghouseApi
```

Comment:
oh, ok, I tried and you don't need it for `deriveRouter` but you do for the `controllers` parameter down of the server.